src/main/java/Duke.java

```diff
@@ -6,6 +6,7 @@ public class Duke {
     public static final String COMMAND_LIST = "list";
     public static final String COMMAND_MARK = "mark";
     public static final String COMMAND_UNMARK = "unmark";
+
     private static TaskList TASKLIST = new TaskList();
```

Comment:
Is it necessary for this variable name to be in all caps? It is not a constant variable.